### PR TITLE
Rollout Update

### PR DESCRIPTION
**Pull request by priyanka-TL**
[Story 1356](https://katha.shikshalokam.org/story-view-1356.html)

### CODE REVIEW

**Review by adithyadinesh0412 — APPROVED** [review]
Approved on 3rd Dec

File: src/services/rollouts.js [hunk 1]
```diff
@@ -114,12 +114,117 @@ module.exports = class RolloutsHelper {
...
+						responseCode: 'CLIENT_ERROR',
+					})
+				}
+			}
+
+			bodyData = _.omit(bodyData, ['id', 'resource_type', 'type', 'organization_id', 'user_id'])
+
+			if (bodyData.targeting_criteria) {
+				const fileName = `${loggedInUserId}${rolloutId}rollout.json`
+				const rolloutUploadStatus = await resourceService.uploadToCloud(
```
Comment (aks30) [inline src/services/rollouts.js:187]: @priyanka-TL can you share sample complete URL in cloud for rollout file and also the resource cloud file that will be stored?

Comment (priyanka-TL) [reply src/services/rollouts.js:187]: rollout/314/28/2b8af968-18a0-4110-a6a1-19b58178bc34/31428rollout.json
this we are storing in db 
complete url : https://mentoring-prod-storage.s3.ap-south-1.amazonaws.com/rollout/314/28/2b8af968-18a0-4110-a6a1-19b58178bc34/31428rollout.json

File: src/constants/interface-routes/configs.json [hunk 2]
```diff
@@ -742,6 +742,32 @@
 					"responseBody": []
 				}
 			]
+		},
+		{
+			"sourceRoute": "/scp/v1/rollouts/update",
```
Comment (aks30) [inline src/constants/interface-routes/configs.json]: @priyanka-TL when will this route be called without id

**Review by aks30 — CHANGES_REQUESTED** [review]
Reviewed on 3rd Dec 11:50 PM

File: src/constants/interface-routes/configs.json (hunk 2 above)
Comment (priyanka-TL) [reply src/constants/interface-routes/configs.json]: for create

**Pull request merged**